frontend/src/components/explorer/mapViewer/DataOverlay.vue

```diff
@@ -211,24 +211,25 @@ export default {
     }),
     filteredDataSourcesIndex() {
       if (this.$route.name === 'interaction-details') {
-        // do not include fluxomics data for the interaction partners page
-        const { fluxomics, ...dataSourcesIndex } = this.dataSourcesIndex;
+        // do not include reactionType data for the interaction partners page
+        const { reactionType, ...dataSourcesIndex } = this.dataSourcesIndex;
         return dataSourcesIndex;
       }
       return this.dataSourcesIndex;
     },
     filteredDataTypes() {
-      // Do not show fluxomics data for the interaction partners page
+      // Do not show reactionType data for the interaction partners page
       // The data type may still be selected, but not shown
       if (this.$route.name === 'interaction-details') {
-        const dataTypes = this.dataTypes.filter(elem => elem.name !== 'fluxomics');
+        const dataTypes = this.dataTypes.filter(elem => elem.name !== 'reactionType');
```

Comment:
The IP page is currently broken (`reaction` still shows up under data type select). It seems like `reactionType` should just be `reaction` here.

```suggestion
        // do not include reaction data for the interaction partners page
        const { reaction, ...dataSourcesIndex } = this.dataSourcesIndex;
        return dataSourcesIndex;
      }
      return this.dataSourcesIndex;
    },
    filteredDataTypes() {
      // Do not show reaction data for the interaction partners page
      // The data type may still be selected, but not shown
      if (this.$route.name === 'interaction-details') {
        const dataTypes = this.dataTypes.filter(elem => elem.name !== 'reaction');
```

Reply:
That's right, after implementing the change with `reactionType` I switched to `reaction` - and it seems I haven't re-tested everything. Thanks for catching this.

Reply:
This was implemented separately through commit [0c3a8fa](https://github.com/MetabolicAtlas/MetabolicAtlas/pull/1181/commits/0c3a8fa625ca66bbc821435b1e09e9a414363057). I had to force-push a couple of times till I got the syntax for `Co-authored-by` right.